apis/go.mod

```diff
@@ -74,3 +75,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/openstack-k8s-operators/nova-operator/api => github.com/gibizer/nova-operator/api v0.0.0-20221129170244-4e3123df52f6
```

Comment:
This depends on https://github.com/openstack-k8s-operators/nova-operator/pull/168